tests/CMakeLists.txt

```diff
@@ -102,9 +103,11 @@ target_include_directories(t_liteclientHSM PRIVATE ${TEST_INCS} ${AKTUALIZR_DIR}
 target_link_libraries(t_liteclientHSM ${TEST_LIBS} uptane_generator_lib testutilities)
 add_dependencies(t_liteclientHSM make_ostree_sysroot)
 set_tests_properties(test_liteclientHSM PROPERTIES LABELS "aklite:liteclientHSM")
+aktualizr_source_file_checks(liteclientHSM_test.cc)
```

Comment:
shouldn't we do the source file check unconditionally? I was thinking that's how things were done in other places.

Reply:
makes sense in terms of consistency. updated.